### ISSUE TITLE
songs not showing unless both in osu and opsu folder

### ISSUE THREAD

**Issue by AmberTek** [yugecin/opsu-dance#263]
I've tried adding multiple songs, but it seems like only a small handful of the ones I originally had for Osu! are actually used. I only have 15 that show up, despite adding far more to the song folder. Do only specific songs work for this?

**Comment by yugecin** [comment]
Since you closed the issue, can you tell how you solved it? It may help other people.
One thing that pops to mind is that opsu can only do the osu!std game mode.

**Comment by AmberTek** [comment]
I had to run the song is Osu! first, then copy the folder to the opsu song folder. If the folder was only in the Osu folder or only in the Opsu folder, it wasn't reading it for some reason. Had to be in both

**Comment by yugecin** [comment]
Hmm that's weird though, it should work when having the songs only in the osu! folder

**Comment by AmberTek** [comment]
Yeah that's what I tried originally. I have over 60 songs in my Osu! folder, but Opsu was only giving me 15. So I tried putting them all in the Opsu folder, but then I had even less. Only showed like 5. But when I copied all the songs to put them back over to Osu! and hence had copies in both folders, it suddenly worked
